packages/nimble-components/src/table/components/row/styles.ts

```diff
@@ -52,14 +53,26 @@ export const styles = css`
         background-color: ${fillHoverSelectedColor};
     }
 
+    .spinner-container,
     .expand-collapse-button {
+        flex: 0 0 auto;
         padding-left: calc(
             ${mediumPadding} + (var(--ni-private-table-row-indent-level) - 1) *
                 ${controlHeight}
         );
     }
 
+    .spinner-container {
+        width: ${controlSlimHeight};
+        height: ${controlSlimHeight};
+        align-self: center;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+    }
```

Comment:
Can we avoid sharing styles in this way to [preserve dom order of selectors](https://github.com/ni/nimble/blob/main/packages/nimble-components/docs/css-guidelines.md#group-selectors-by-target-and-document-order)?
If we want to avoid duplication of that padding styling we can do so via private css properties.